odoo/tools/config.py

```diff
@@ -470,6 +473,12 @@ def die(cond, msg):
                     os.path.abspath(os.path.expanduser(os.path.expandvars(x.strip())))
                       for x in self.options['addons_path'].split(','))
 
+        self.options['migrations_path'] = (
```

Comment:
I think it would be useful to allow multiple `upgrades_path` entries, in the same fashion as `addons_path`, and it does not cost much to support it.